alphagov/paas-admin
# Update to govuk-frontend v4.0.0 and make required changes

What
----

Describe what you have changed and why.

- update dependency
- make changes where required from https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0

How to review
-------------

- checkout, update dependency and run 
- check commits for details
- check pages render ok

Who can review
---------------

not @kr8n3r 
---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
